pkg/sql/lex/allkeywords/allkeywords.go

```diff
@@ -22,6 +20,7 @@
 // (~65k entries) that the mph package never returned from its Build call.
 //
 // A `KeywordsTokens = map[string]int32` map from string -> token id.
+// +build all-keywords
```

Comment:
this needs to be where it originally was

Reply:
If we move it as it was originally, bazel is failing due to the same error:`external/go_sdk/pkg/tool/darwin_amd64/link: /private/var/tmp/_bazel_admin/e227da31e102dc39aa010ed5c3f3a075/sandbox/darwin-sandbox/18164/execroot/cockroach/bazel-out/host/bin/pkg/sql/lex/all_keywords.a: not package main`

Reply:
ah ok, yeah just remove the line. it is in a separate package so it doesn't matter that we have this tag.

note putting `// +build all-keywords` here is a documentation comment on this package, rather than a directive.

Reply:
Wait, if I remove the new line in line (right after `// +build all-keywords`) code is working in both (bazel and make) so code looks something like:
```
// licenses/APL.txt.

// +build all-keywords
// all-keywords generates sql/lex/keywords.go from sql.y.
//
// It is generically structured with Go templates to allow for quick
// prototyping of different code generation structures for keyword token
// lookup. Previous attempts:
//
// Using github.com/cespare/mph to generate a perfect hash function. Was 10%
// slower. Also attempted to populate the mph.Table with a sparse array where
// the index correlated to the token id. This generated such a large array
// (~65k entries) that the mph package never returned from its Build call.
//
// A `KeywordsTokens = map[string]int32` map from string -> token id.
package main
```

BUT if I add the new line again in line 12 (as it is in master now) is failing.

What do you think?

Reply:
the former works that's because `// +build all-keywords` is part of the package comment.

```
// anything here
// is a comment
package main
```

but putting any spacing in between makes it a directive, i.e.
```
// +build thisisapackage

// this is a comment
package main
```

which is why latter doesn't work.

we don't need this tag anymore because we have it in a separate package. we needed it originally as there is a main and non-main in the same package, and we didn't want all_keywords.go to build as a main in that package. since it is a separate package which does not build by default, we no longer need it.